gdk-sys/build.rs

```diff
@@ -92,3 +98,19 @@ fn find() -> Result<(), Error> {
         Err(err) => Err(err),
     }
 }
+
+#[cfg(not(feature = "dox"))]
+fn check_features() {
+    // The pkg-config file defines a `targets` variable listing the
+    // various backends that gdk was compiled for.
+    // We extract that and create gdk_backend="x11" and the like
+    // as configuration variables.
+    // For reference, the backend set at time of writing consists of:
+    // x11 win32 quartz broadway wayland
+    if let Ok(targets) = pkg_config::get_variable(PKG_CONFIG_PACKAGE, "targets") {
+        println!("cargo:backends={}", targets);
```

Comment:
What is this doing? Shouldn't it be `cargo:rustc-cfg=backends={}`?

Reply:
https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key

Reply:
No, this is exporting a dependency variable for crates which depend on us to use.  In dependent crates' `build.rs` the environment variable `DEP_GDK_BACKENDS` will be set

Reply:
Not `DEP_GDK_SYS_BACKENDS`? Where does the `GDK` part come from if not the crate name?

Reply:
Cargo strips the -sys -- don't ask me how :D

Reply:
As says doc I linked, "gdk" come from `links="gdk"` in Cargo.toml

Reply:
Makes sense to me then

Reply:
Maybe update previous comment with something like "Backends list can be used in build script of immediate dependent package as environment variable  `DEP_GDK_BACKEND`" ?

Reply:
@EPashkin Good idea, I've pushed an update.